crypto/arm64cpuid.pl

```diff
@@ -25,7 +25,8 @@
 #include "arm_arch.h"
 
 .text
-.arch	armv8-a+crypto
+.arch   armv8.2-a+crypto
+.arch_extension sha3
```

Comment:
what does this change do?

Reply:
This tells the assembler that we're targeting CPUs with the SHA3 extension, which is where the EOR3 instruction is found. Without this the assembler wouldn't accept the instructions (SHA3 extension is an optional extension in the Armv8.2-A architecture specification)

Reply:
(Not strictly necessary here as the instruction is manually encoded)

Reply:
Yeah, that's what I mean, when it's not really needed,
it's safer to remove it.

Reply:
It's manually encoded, not very necessary, will remove it in next commit.